sources/packages/web/src/views/aest/AppAEST.vue

```diff
@@ -1,45 +1,61 @@
 <template>
   <IdleTimeChecker :clientIdType="ClientIdType.AEST">
-    <v-app-bar dense flat app style="overflow: visible">
-      <BCLogo
+    <v-app-bar color="white">
+      <b-c-logo
         subtitle="Ministry of Advanced Education and Skills Training"
-      ></BCLogo>
-      <v-spacer></v-spacer>
-      <check-permission-role :role="Role.AESTCreateInstitution">
-        <template #="{ notAllowed }">
-          <v-btn
-            v-if="isAuthenticated"
-            variant="text"
-            prepend-icon="fa:fa fa-edit"
-            :disabled="notAllowed"
-            @click="
-              $router.push({ name: AESTRoutesConst.INSTITUTION_PROFILE_CREATE })
-            "
-            >Create institution</v-btn
-          >
-        </template>
-      </check-permission-role>
-      <v-menu v-if="isAuthenticated">
-        <template v-slot:activator="{ props }">
-          <v-btn
-            class="mr-5"
-            icon="fa:fa fa-user"
-            variant="outlined"
-            elevation="1"
-            color="secondary"
-            v-bind="props"
-          ></v-btn>
-        </template>
-        <v-list>
-          <v-list-item :value="menuItem.label">
-            <v-list-item-title>
-              <v-list-item-title @click="menuItem.command">
-                <span class="label-bold">{{ menuItem.label }}</span>
+      ></b-c-logo>
+      <v-btn-toggle
+        selected-class="active-btn label-bold"
+        v-model="toggleNav"
+        class="navigation-btn float-left"
+      >
+        <check-permission-role :role="Role.AESTCreateInstitution">
+          <template #="{ notAllowed }">
+            <v-btn
+              v-if="isAuthenticated"
+              class="nav-item-label"
+              variant="text"
+              :to="{
+                name: AESTRoutesConst.AEST_DASHBOARD,
+              }"
+              >Home</v-btn
```

Comment:
just curious. Does Home need an icon?? `Create institution` has one. maybe you can check with @hellolynn-tbtb  when she is back

Reply:
I think @hellolynn-tbtb can answer. Please respond to this.

Reply:
@hellolynn-tbtb is on vacation, either Figma has it and it should be added, or does not have it and it should not be added.
If the button is not in Figma and @hellolynn-tbtb asked to added during a conversation without further directions, please follow the other "Home" button in the application as @ann-aot mentioned.